Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -245,7 +245,8 @@ open class ChatMessageListVC: _ViewController,
             // Because we use an inverted table view, we need to avoid animations since they look odd.
             UIView.performWithoutAnimation {
                 listView.updateMessages(with: changes) { [weak self] in
-                    if let newMessageInserted = changes.first(where: { $0.isInsertion && $0.indexPath.row == 0 })?.item {
+                    
+                    if let newMessageInserted = changes.first(where: { ($0.isInsertion || $0.isMoved) && $0.indexPath.row == 0 })?.item {
```

Comment:
@alteral this change should fix the bug when `giphy` message sent with the delay does not cause scrolling to bottom 👏

Reply:
Why is this part of this PR tho? 🤔 This issue is tracked here: https://stream-io.atlassian.net/browse/CIS-1600. Can we move this to another PR, please? So that it is easy to track it and update the changelog accordingly.

Reply:
This is needed for this PR so moving it to another one would leave this feature incomplete.

Reply:
I will open a new PR so that CIS-1600 can get solved with the intention of being merged before this one.